src/assets/data/dishTypes.json

```diff
@@ -23,5 +23,20 @@
         "id": 4,
         "name": "Desserts",
         "slug": "desserts"
+    },
+    {
+        "id": 5,
```

Comment:
Remove this last three categories as they don't belong on dish courses

Reply:
Resolved